requirements.txt

```diff
@@ -1,70 +1,4 @@
-addict==2.2.0
-amqp>=2.3.2
-asn1crypto==0.24.0
-astroid==2.0.4
-attrs==18.2.0
-avro-cwl==1.8.4
-bagit==1.7.0
-billiard>=3.5.0.4
-CacheControl==0.11.7
-celery==5.2.2
-certifi==2018.8.24
-cffi>=1.11.5
-chardet==3.0.4
-click>=6.7
-clickclick==1.2.2
-connexion==1.5.2
-cryptography==3.3.2
-decorator==4.3.0
-Flask==1.0.2
-Flask-Cors==3.0.9
-Flask-PyMongo==2.1.0
-future==0.16.0
-gunicorn==19.9.0
-html5lib==1.0.1
-idna==2.7
-inflection==0.3.1
-isodate==0.6.0
-isort==4.3.4
-itsdangerous==0.24
-Jinja2==2.11.3
-jsonschema==2.6.0
-kombu>=4.2.1
-lazy-object-proxy==1.3.1
-lockfile==0.12.2
-lxml==4.6.5
-MarkupSafe==1.1.1
-mccabe==0.6.1
-mistune==0.8.1
-mypy-extensions==0.4.1
-networkx==2.2
-prov==1.5.1
-psutil==5.6.6
-py-tes==0.3.0
-pycparser==2.19
-PyJWT==2.4.0
-pylint==2.1.1
-pymongo==3.7.1
-pyparsing==2.2.1
-python-dateutil==2.6.1
-pytz==2018.5
-PyYAML==5.4
-rdflib==4.2.2
-rdflib-jsonld==0.4.0
-requests>=2.20.0
-ruamel.yaml==0.15.51
-scandir==1.9.0
-schema-salad==3.0.20181129082112
-shellescape==3.4.1
-simplejson==3.16.0
-six==1.11.0
-subprocess32==3.5.2
-swagger-spec-validator==2.3.1
-#-e git+https://github.com/elixir-europe/TEStribute.git#egg=testribute
-typed-ast==1.1.0
-typing==3.6.6
-typing-extensions==3.6.5
-urllib3>=1.26.5
-vine>=1.1.4
-Werkzeug==0.15.3
-wrapt==1.10.11
+foca==0.8.0
```

Comment:
Please use the [latest version of FOCA](https://github.com/elixir-cloud-aai/foca/releases/tag/v0.9.0) and adapt your codebase accordingly.

Reply:
The latest version is 0.9.0

Reply:
Done.